1-js/05-data-types/01-primitives-methods/1-string-new-property/solution.md

```diff
@@ -9,10 +9,10 @@ str.test = 5; // (*)
 alert(str.test); 
 ```
 
-这里有两种结果：
+取决于你是否启用严格模式，结果可能是：
```

Comment:
Depending on whether you have `use strict` or not, the result may be:
1. `undefined` (no strict mode)
2. An error (strict mode).
```suggestion
取决于你是否启用严格模式（`use strict`），结果可能是：
```